README.md

```diff
@@ -1,2 +1,116 @@
-# community
-CoreDash Governance
+# Welcome to the CoreDash Community!
+
+With the recent license changes of the Grafana project, there are quite a few
+community end users as well as monitoring vendors who are interested in a
+PromQL visualization and dashboarding solution that is under the Apache-2.0 
```

Comment:
mmm I think the initial thought of this group was to have tools around PromQL to reduce the time required to learn this language.

Visualization is one part, but then you will likely have some tools to debug PromQL (like PromLens is doing for example). And perhaps at some point, some nice interactive tutorial to learn Javascript. (Thought like you can have in the video games, would be so cool).
I'm thinking also having Prometheus Alerting sandbox. Where you enter your alert and it show you when it would have been fired. (I know it's an opened issue on Prometheus)

Probably it's not so odd to put it as the main target.

Reply:
Reading the initial (private) document again, it indeed focuses on PromQL ("To give a new project the best chance of being successful, reducing the initial scope to provide faster value for end users is ideal from our perspective. Therefore, having a visualization project that is focused on Prometheus and PromQL seems to be a logical starting point."). It uses a very similar wording for the rest of the scope as here in this PR... but I had a feeling like there was enough of an agreement in the actual meetings that whatever we build should just start with PromQL for practical purposes (to have something to show), but then also expand to other datasources.

Personally, I'd like to see a viable open-governance dashboard builder for a multitude of popular datasources. But of course it should have great support for PromQL.

Reply:
These are all excellent points, and I definitely think that the scope and goals sections still need work. I removed the mention of PromQL in the initial project description to make it a little more general, but I left the details about PromQL in the scope / goals sections. I also made it more clear that the scope and goals are still in draft mode and are likely to change. I think this is probably good enough for the initial versions of the page, but maybe we want to devote some time in a meeting or elsewhere to iterate on them shortly after we go public. Does this sound like a reasonable approach?

Reply:
Fine for me. Thanks for the update !